docker/Dockerfile.linux

```
@@ -1,21 +1,43 @@
-FROM mono:6.12
-
-MAINTAINER Justin Phelps <linuturk@onitato.com>
+ARG monoversion="6.12"
+FROM mono:${monoversion} as build
 
 RUN apt-get update && apt-get install git -y
 
 COPY . /usr/local/src/choco/
 
 WORKDIR /usr/local/src/choco
-RUN chmod +x build.sh
-RUN chmod +x zip.sh
-RUN ./build.sh
+RUN chmod +x *.sh
+
+ARG buildscript="build.sh"
+RUN ./$buildscript
+
+RUN if [ "$buildscript" = "build.official.sh" ]; then \
+    cp docker/choco_official_wrapper code_drop/chocolatey/console/choco_wrapper; \
+  else \
+    cp docker/choco_wrapper code_drop/chocolatey/console/choco_wrapper; \    
+  fi;
+
+
+ARG monoversion="6.12"
+FROM mono:${monoversion} as install
+
+ARG monoversion="6.12"
+LABEL org.opencontainers.image.url="https://chocolatey.org/"
+LABEL org.opencontainers.image.documentation="https://docs.chocolatey.org/"
+LABEL org.opencontainers.image.source="https://github.com/chocolatey/choco"
+LABEL org.opencontainers.image.vendor="Chocolatey Software, Inc"
+LABEL org.opencontainers.image.licenses="Apache-2.0"
+LABEL org.opencontainers.image.title="Chocolatey"
+LABEL org.opencontainers.image.description="Chocolatey Client running on Mono"
+LABEL org.opencontainers.image.authors="https://chocolatey.org/support/"
```

```suggestion
LABEL org.opencontainers.image.authors="https://chocolatey.org/support"
```
Otherwise, it goes to a 404

---

Whoops, I screwed that one up. Fixed.